piff/gp_interp.py

```diff
@@ -0,0 +1,324 @@
+# Copyright (c) 2016 by Mike Jarvis and the other collaborators on GitHub at
+# https://github.com/rmjarvis/Piff  All rights reserved.
+#
+# Piff is free software: Redistribution and use in source and binary forms
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+"""
+.. module:: sklearn_gp_interp
+"""
+
+from .interp import Interp
+from .star import Star, StarFit
+
+import numpy as np
+
+class GPInterp(Interp):
+    """
+    An interpolator that uses sklearn.gaussian_process to interpolate a single surface.
+
+    :param  attr_interp: A list of star attributes to interpolate from
```

Comment:
I think I prefer the name `keys` for this, but if people prefer `attr_interp` that's ok too.  I mostly just want the same name for the same concept across different classes.